app/component/search/search-modal.cjsx

```diff
@@ -36,40 +25,20 @@ class SearchModal extends React.Component
     <div style={style}
       className="search-modal">
       <div className="row">
-        <div className="small-12 medium-6 medium-offset-3 columns cursor-pointer search-header" onClick={@closeModal}>
-          <Icon img={'icon-icon_arrow-left'} />
-          <span className="search-header-separator"/>
+        <div className="small-12 medium-6 medium-offset-3 columns cursor-pointer search-header">
+          <span className="search-header__back-arrow" onClick={@props.closeModal}>
+            <Icon img={'icon-icon_arrow-left'}/>
+            <span className="search-header-separator"/>
+          </span>
+          <Tabs
+            inkBarStyle={{display: "none"}}
+            tabItemContainerStyle={{backgroundColor: "#eef1f3", lineHeight: "18px", marginLeft: "28px", width: "calc(100% - 28px)"}}
```

Comment:
Only override is provided by material-ui. They do not provide className for that. We could find the div and that way override line-height and margin-left. But the others we could not override without !Important.